ScanCommand/tool-scan/Package.md

```diff
@@ -0,0 +1 @@
+Ballerina Scan tool
```

Comment:
Tools might get listed in Ballerina Central in future. In that case this will be used to display the tool details. So let's have an adequate description here.